libs/typescript/ts-utils/common-utils.ts

```diff
@@ -148,13 +148,37 @@ export function assertNotNull<T>(
   return value;
 }
 
+export function checkIfNull(value: string | null | undefined, msg: string) {
+  if (
+    value === null ||
+    value === undefined ||
+    (typeof value === 'string' && !value.length)
+  ) {
+    console.log(msg);
+    return true;
+  }
+  return false;
+}
+
```

Comment:
We already have such a function above. Why add a new one?

Reply:
That allows me to create getSecretEnvString where we only warn(not throw error) when we see a missing ENV